docs/topics/data-analysis/kotlin-notebook-add-dependencies.md

```diff
@@ -22,6 +22,10 @@ However, Kotlin Notebook has a simplified method to load popular libraries in th
 ```kotlin
 // Replace libraryName with the library dependency you want to add
 %use libraryName
+// You may specify a version
```

Comment:
Let's add that you may use optional named parameter `v=` and invoke completion to check available versions

Reply:
https://github.com/martinbonnin/kotlin-web-site/commit/021f13d6787ce996c03fe06da510abbf92ceba91